src/main/java/entity/Person.kt

```diff
@@ -0,0 +1,27 @@
+package entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Transient
+
+@Entity
+class Person (
```

Comment:
```suggestion
@Table(name = "users")
@Entity
class Person (
```

요구사항 3 에 추가 된  `@Table(name = "users")` 부분도 추가 되어야 하는데 누락된 것 같아요 😅